app/src/main/res/values/strings.xml

```diff
@@ -45,6 +45,7 @@
     </plurals>
     <string name="text_tools_all_installed">You have downloaded all available tools.</string>
     <string name="menu_add_to_home">Add to Home screen</string>
+    <string name="action_tools_open">open tool</string>
```

Comment:
Should this be `Open Tool` or does case not matter?

Reply:
Ann already changed this upstream, next time we sync translations from OneSky this should be fixed

Reply:
my 1 minor nit for this line is to move it up with the other actions for the tools UI (lines 35-36)